src/ServiceControl.Persistence.RavenDB/EmbeddedDatabase.cs

```diff
@@ -19,12 +18,10 @@ namespace ServiceControl.Persistence.RavenDB
 
     public sealed class EmbeddedDatabase : IDisposable
     {
-        EmbeddedDatabase(RavenPersisterSettings configuration, IHostApplicationLifetime lifetime)
+        EmbeddedDatabase(RavenPersisterSettings configuration)
```

Comment:
Given we are using an `IHostedService` I don't see a need for having any knowledge of `IHostApplicationLifetime`